lib/pharos/configuration/host.rb

```diff
@@ -42,9 +40,9 @@ def to_s
         # @return [Boolean]
         def overlaps?(cidr)
           # special-case the default route and ignore it
-          return nil if @prefix == 'default'
+          return nil if prefix == 'default'
```

Comment:
Rubocop wants you to use both `prefix` and `self.prefix` accessors as well as the `prefix` local in the same function, which is IMO pretty bad.

```
lib/pharos/configuration/host.rb:43:25: C: Style/RedundantSelf: Redundant self detected.
          return nil if self.prefix == 'default'
                        ^^^^^^^^^^^
```

Reply:
Consider renaming the local variable? It's confusing to have exactly the same name.